test/bin/shell-completion.spec.js

```diff
@@ -1,70 +1,22 @@
-const path = require('path');
 const assert = require('chai').assert;
+const { exec } = require('child_process');
 
 describe('shell-completion', () => {
 
-  it('should supply --options and actions', () => {
+  it('should complete a partial command', () => {
 
-    // expect
-    return assertShellComplete('',
-        '--instance',
-        '--local',
-        '--url',
-        '--user',
+    let partial = 'comp';
+    let bin = 'medic-conf';
+    let input = bin + ' ' + partial;
 
-        '--help',
-        '--shell-completion',
-        '--supported-actions',
-        '--version',
-        '--accept-self-signed-certs',
+    let cmd = `bash -i -c 'source /etc/bash_completion; COMP_LINE="${input}"; COMP_WORDS=(${input}); COMP_CWORD=1; COMP_POINT=${input.length}; $(complete -p medic-conf | sed "s/.*-F \\([^ ]*\\) .*/\\1/") && echo \$\{COMPREPLY[*]\}'`; // eslint-disable-line no-useless-escape
```

Comment:
Consider making this a multi-line string.
I don't really understand what it does, so maybe some comments.

Reply:
Updated